applications/smart-distancing/face_mask_recognition/inference.py

```diff
@@ -0,0 +1,78 @@
+import tensorflow as tf
+import numpy as np
+from PIL import Image
+import matplotlib.pyplot as plt
+import cv2
+from absl import flags
+import os
+
+flags.DEFINE_string("model_dir", "inference_model", "Directory of .pb file")
+
+flags.DEFINE_integer("input_size", 224, "The size of input image")
+
+flags.DEFINE_string("imgs_dir", "images", "Directroy of images for inference")
+
+flags.DEFINE_list("classes", ["face", "face-mask"], "List of classes names")
+
+flags.DEFINE_string("result_dir", "results", "Exported images path")
+
+FLAGS = tf.flags.FLAGS
+
+
+def main(_):
+    face_mask_graph = tf.Graph()
```

Comment:
If you use TensorFlow V2 or `tf.compat.v2` API it will be much easier to get inference from a saved model with running followings:
```
model = tf.saved_model.load(str(model_dir))
model = model.signatures['serving_default']
input_image = np.expand_dims(resized_rgb_image, axis=0)
input_tensor = tf.convert_to_tensor(input_image)
output_dict = self.detection_model(input_tensor)
```

Reply:
I should test it. I remember there were some issues with tf.compat.v2 and tf2 which pushed me to use this approach. However, I will check it again.